about.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>ICT4D: ICT4D for Development of Africa</title>
+  <link rel="stylesheet" href="./css/about.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <div class="wrapper">
+    <header class="header1">
+      <ul class="little-header-list">
+        <li class="list-item">
+          <a href="#"> <i class="fa fa-facebook"></i> </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> <i class="fa fa-twitter"></i> </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> English </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> My Page </a>
+        </li>
+        <li class="list-item">
+          <a href="#"> Logout </a>
+        </li>
+      </ul>
+
+    </header>
+    <header class="header2">
+      <div class="menu-bar-container">
+        <ul class="menu-bar">
+          <li>
+           <a href="#" class="hamburger">
+            <i class="fa fa-bars menu-bar-icon"></i>
+           </a>
+          </li>
+          <li>
+            <a href="#" class="close-icon"> <i class="fa fa-times"></i> </a>
+          </li>
+        </ul>
+        <div class="menu-image menu-logo-mobile">
+          <a href="./index.html"> <img src="./images/logo.png" alt="Logo Image"> </a>
+        </div>
+      </div>
+      <nav class="navbar">
+        <div class="menu-image menu-image-desk">
+          <a href="./index.html"> <img src="./images/logo.png" alt="Logo Image"> </a>
+        </div>
+        <ul class="menu">
+          <li class="menu-item">
+            <a  class="menu-item-link" href="./index.html"> Home </a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-item-link" href="#agendas"> Past Global Summit </a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-item-link" href="#partner-id"> Partners </a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-item-link" href="./about.html"> About </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+  
+    <section id="home"  class="section-conference-invitation section-wrapper">
```

Comment:
- [ ] Here again kindly follow the given design. Fix the `h1` heading that is currently missing, 
- [ ] Center all the text for this section
- [ ] Make the text which is `bold` to be `bold`
- [ ] Wrap the email address in an `a` tag as it is the semanric tag.

Your design
![image](https://user-images.githubusercontent.com/81932478/191734564-08b56a90-edd2-45c9-8288-7aa7984757fb.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/191734682-a98160e2-fa65-4b63-b530-91b8e153fb0e.png)